Sources/SnowManLogic/ViewLogic/MockView.swift

```diff
@@ -0,0 +1,40 @@
+public class MockView: Viewable {
+	var ui: Receivable 
+	var guess: String!
+
+	public init(ui: Receivable) {
+		self.ui = ui
+	}
+
+	public func receiveGuess() -> String {
+		guess = ui.getUserInput()
+		if (containsOnlyLetters(guess)) {
+			return guess
+		} else {
+			return "Invalid"
+		}
+	}
+
+	public func assignBlanks(gameWord: String, correctGuesses: [String]) -> String {
+		var blanks: [String] = []
+		let letters = gameWord.characters.map { String($0) }
+		
+		for letter in letters { 
+			if (correctGuesses.contains(letter)) { 
+           		blanks.append(letter)  
+       		} else { 
+           		blanks.append("__ ") 
+       		} 
+   		}
+   		return blanks.joinWithSeparator(" ")
+	}
+
+	private func containsOnlyLetters(input: String) -> Bool {
```

Comment:
the logic here is basically the same as in the implementation, the mock/stub object should be void of logic.